providers/os/registry/registrykey_powershell.go

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package registry
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+)
+
+func ParsePowershellRegistryKeyItems(r io.Reader) ([]RegistryKeyItem, error) {
+	data, err := io.ReadAll(r)
+	if err != nil {
+		return nil, err
+	}
+
+	var items []RegistryKeyItem
+	err = json.Unmarshal(data, &items)
+	if err != nil {
+		return nil, err
+	}
+
+	return items, nil
```

Comment:
nit; there is no need to check if err is nil
```suggestion
	return items, err
```